docs/deploys.md

```diff
@@ -0,0 +1,120 @@
+# Halyard Deploys
+
+## Purpose
+
+Explain how Halyard handles deploying Spinnaker.
+
+## Terminology
+
+* **Deployment Type**: How Spinnaker is divided among all the instances it is
+ running on. An example is having each Spinnaker service run a separate
+ instance. See [Deployment Types](#deployment-types) section for more
+ examples.
+* **Service Dependencies**: Dependencies of Spinnaker that need to be run as
+ stand-alone services. An example is Redis, a non-example is Packer.
+* **Deployment Environment**: The infrastructure Spinnaker is running on,
+ Spinnaker's Deployment Type, and Spinnaker's service dependencies.
+
+## Deployment Types
+
+* **LocalhostDebian**: This is every Spinnaker service installed as a Debian
+ package on the node Halyard is running on.
+* **LocalhostGit**: This is every Spinnaker service installed from Git on the 
+ node Halyard is running on. This is intended for use by Developers.
+* **Flotilla**: This is every Spinnaker service installed remotely on a
```

Comment:
You're right, I see that "distributed" makes sense over "flotilla". But I don't think "standalone" correctly conveys the sentiment that Spinnaker is being installed locally. I think just "local" makes more sense, and doesn't conflate with loopback. So we'd have

LocalDebian, LocalGit, and Distributed